contributing/notebook_template/notebook_template.ipynb

```diff
@@ -122,11 +125,13 @@
     }
    },
    "source": [
-    "### Loading Data\n",
+    "## Querying for TESS Data (Rename)\n",
```

Comment:
This could be
### Loading data (Raname, e.g. Querying for TESS Data)

I think it should get the Subsection type instead of the Section type since it goes under Main Content.